packages/salesforcedx-vscode-soql/test/vscode-integration/editor/soqlEditorInstance.test.ts

```diff
@@ -136,22 +145,64 @@ describe('SoqlEditorInstance should', () => {
 
     assert(postMessageSpy.calledWith(expectedMessage));
   });
+
+  it('handles query event and updates text document with soql', async () => {
+    const updateDocumentSpy = sandbox.spy(instance, 'updateTextDocument');
+    instance.sendEvent({
+      type: MessageType.QUERY,
+      message: JSON.stringify(uiModelOne)
+    });
+    assert(
+      updateDocumentSpy.callCount === 1,
+      `updateDocumentSpy callcount expected 1, but got ${
+        updateDocumentSpy.callCount
+      }`
+    );
+    assert(
+      updateDocumentSpy.getCall(0).args[1].indexOf(uiModelOne.sObject) > -1,
+      `updateDocumentSpy was called with ${
+        updateDocumentSpy.getCall(0).args[1]
+      } but does not include ${uiModelOne.sObject}`
+    );
+  });
+
+  it('handles activation event and updates the webview', async () => {
+    const updateWebviewSpy = sandbox.spy(instance, 'updateWebview');
+    instance.sendEvent({
+      type: MessageType.ACTIVATED
+    });
+    assert(
```

Comment:
can we use `expect` instead of `assert` to make this validation more readable? similar to what we have in soqlUtils.test.ts.

Reply:
Done